src/Symfony/Component/Routing/RequestContext.php

```diff
@@ -81,7 +81,7 @@ public function getBaseUrl()
      *
      * @return $this
      */
-    public function setBaseUrl($baseUrl)
+    public function setBaseUrl(string $baseUrl)
```

Comment:
you are missing some types in this class, e.g. in setParameter, setHttpsPort. I also think setQueryString should add a string type. If it accepts null, the getter should also return null but it doesn't. So I would suggest to just drop accepting null which also wasn't documented in phpdoc. The broken test for this can be removed then.

Reply:
Ok since there are more nullable cases like this in the Route class, let's just keep the nullable behavior and make it `setQueryString(?string)`

Reply:
You‘re right, adding the type-hints that I forgot. And yes, there is actually one unit test that calls `setQueryString(null)`, so I‘m making it nullable.